Scripts/SanePdfReport/SanePdfReport.py

```diff
@@ -0,0 +1,82 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import traceback
+
+import os
+import base64
+import random
+import string
+import subprocess
+from pathlib import Path
+
+WORKING_DIR = Path("/app")
+INPUT_FILE_PATH = 'sample.json'
+OUTPUT_FILE_PATH = 'out{id}.pdf'
+DISABLE_LOGOS = True  # Bugfix before sane-reports can work with image files.
+
+
+def fix_base64(base64str, filename):
+    """ Some of the base64 strings are too big for the shell, so let's write
+    them to disk """
+    with open(WORKING_DIR / filename, "wb") as fh:
+        base64str = base64str[base64str.index(";base64,") + 8:]
+        fh.write(base64.b64decode(base64str))
+
+
+def random_string(size=10):
+    return ''.join(
+        random.choices(string.ascii_uppercase + string.digits, k=size))
+
+
+try:
+    sane_json_b64 = demisto.args().get('sane_pdf_report_base64', '').encode(
+        'utf-8')
+    orientation = demisto.args().get('orientation', 'portrait')
+    resourceTimeout = demisto.args().get('resourceTimeout', '4000')
+    reportType = demisto.args().get('reportType', 'pdf')
+    headerLeftImage = demisto.args().get('customerLogo', '')
+    headerRightImage = demisto.args().get('demistoLogo', '')
+    pageSize = demisto.args().get('paperSize', 'letter')
+    disableHeaders = demisto.args().get('disableHeaders', '')
+
+    if DISABLE_LOGOS:
```

Comment:
Do we have an option that it won't be true?

Reply:
When we fix the main repo (sane-reports).

Reply:
please fix sane reports before or ignore the bug... there is no reason to add this const or code to address that

Reply:
> please fix sane reports before or ignore the bug... there is no reason to add this const or code to address that

This literally ignores the bug. What did you have in mind? 
@hirshbergshachar are logos in the reports a must right now or can they wait for a future PR in `sane-reports`?

Reply:
@agamm it can wait for future PR

Reply:
@Bars92 agreed?